api/integrations/grafana/mappers.py

```diff
@@ -0,0 +1,72 @@
+from audit.models import AuditLog
+from audit.services import get_audited_instance_from_audit_log_record
+from features.models import (
+    Feature,
+    FeatureSegment,
+    FeatureState,
+    FeatureStateValue,
+)
+from integrations.grafana.types import GrafanaAnnotation
+from segments.models import Segment
+
+
+def _get_feature_tags(
+    feature: Feature,
+) -> list[str]:
+    return list(feature.tags.values_list("label", flat=True))
```

Comment:
Is there a reason to use `"label"` here instead of `"tag"` - shouldn't we be consistent with the terminology that we use in the Flagsmith platform?

Reply:
It's implied that the function result is a list of Grafana tags.

Reply:
Right, so we don't want to use `"tag"` again because it would be a tag within a tag? 🤔

Reply:
https://github.com/Flagsmith/flagsmith/blob/8582a661fb5a43ab230deeca796047c92bc27314/api/projects/tags/models.py#L13-L14

Reply:
Sorry, brain not switched on yet today. Read this totally wrong! 

So, I guess my follow up question would be whether we need to namespace these at all like we are with the other Grafana tags?

Reply:
Good question. I think As a user, I'd be annoyed by having to prefix those.

Reply:
Fair enough 👍